apps/mobile/src/features/receive/receive-sheets/select-account.tsx

```diff
@@ -30,7 +32,9 @@ export function SelectAccount() {
         />
       }
     >
-      <AccountList accounts={accounts.list} onPress={onSelectAccount} showWalletInfo />
+      <ScrollView>
+        <AccountList accounts={accounts.list} onPress={onSelectAccount} showWalletInfo />
```

Comment:
Just a thought, but should the `ScrollView` perhaps be in the render of `AccountList`? If i see `SomeList` I kind of assume it's a scrollable element in the view.